proxy-robots-module/src/main/java/orwell/proxy/config/ConfigScout.java

```diff
@@ -0,0 +1,56 @@
+package orwell.proxy.config;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlElement;
+import javax.xml.bind.annotation.XmlType;
+
+@XmlType(propOrder = {"camera", "image"})
+public class ConfigScout implements IConfigRobot {
```

Comment:
This is indeed not used so far in the program, only in the tests. It is a way to test  RobotFactory by introducing a new type of robot (the Scout, which could be a different robot than a LegoTank, with different abilities, like by being faster, and using a different wireless protocol). Check ConfigurationTest.xml file in test/resources for a sketch of the idea.